data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -109,6 +104,6 @@ def test_get_output_file_path(self, mock_etl, mock_paths):
 
         output_file_path = etl._get_output_file_path()
         expected_output_file_path = (
-            data_path / "dataset" / "national_risk_index_2020" / "usa.csv"
+            data_path / "dataset" / "national_risk_index" / "usa.csv"
```

Comment:
Is this a breaking change for running the full ETL and score process?

Reply:
No, we run the full ETL on each PR commit and it didn't fail, nor the tests.